tests/FakeItEasy.IntegrationTests/FakingClassesTests.cs

```diff
@@ -12,10 +11,7 @@ public class FakingClassesTests
         [Test]
         public void Should_be_able_to_get_a_fake_value_of_uri_type()
         {
-            using (Fake.CreateScope(new NullFakeObjectContainer()))
-            {
-                A.Fake<Uri>();
-            }
+            A.Fake<Uri>();

```

Comment:
My guess is that the lack of blowing up indicated successful creation of the Fake.
We could assert that it's not null. Or that it's a Fake.
Unfortunately, we don't have any clue about why `Uri`s are interesting.